Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -624,9 +630,21 @@ function New-LDAPSIdentitySource {
             throw "File to certificate conversion failed. See error message for more details"
         }
     }
+    # check if the certicates expire or not
+    foreach ($cert in $Certificates) {
+        $certDate = Get-Date $cert.GetExpirationDateString()
```

Comment:
This should be X509Certificate2 object, which has `NotBefore` and [`NotAfter`](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificate2.notafter?view=net-8.0) properties. We should check the current date falls between these 2.